src/app-nordigen/services/nordigen-service.js

```diff
@@ -208,6 +209,7 @@ export const nordigenService = {
     );
 
     return {
+      iban: accountMetadata.iban,
```

Comment:
I know that based on the Nordigen swagger doc it should be there but as I know not all banks use `iban` some of them use `bban`. Maybe it's somewhere there too and we can use it as a fallback.
related comment - https://github.com/actualbudget/actual/pull/766/files#r1139983083

Reply:
The account metadata endpoint doesn't include `bban` (https://nordigen.com/en/docs/account-information/integration/parameters-and-responses/#/accounts/retrieve%20account%20metadata), account details endpoint does but that endpoint is affected by bank rate limits so it's heavily limited (it also has longer response time). I think endpoints are rate limited separately so I guess it may not really be a problem since we'll be requesting balance and transactions data at the same time.